test/blackbox-tests/test-cases/virtual-libraries/implements-external.t/run.t

```diff
@@ -25,3 +25,25 @@ Implement external virtual libraries with private modules
   $ env OCAMLPATH=vlib/_build/install/default/lib dune build --root impl-private-module --debug-dependency-path
   Entering directory 'impl-private-module'
   Name: implement virtual module. Magic number: 42
+
+Now we install an implementation, and make sure that it remains usable
+  $ export OCAMLPATH=$PWD/vlib/_build/install/default/lib:$PWD/impl/_build/install/default/lib
+  $ mkdir use-external-impl && cd use-external-impl
+  $ cat >dune-project <<EOF
+  > (lang dune 3.0)
+  > EOF
+  $ cat >dune <<EOF
+  > (executable
+  >  (name blah)
+  >  (libraries dune-vlib.impl))
+  > EOF
+  $ cat >blah.ml <<EOF
+  > Vlib.Foo.run ()
+  > EOF
+  $ dune exec ./blah.exe
```

Comment:
Could you add a comment saying what is expected and whether Dune's current behaviour is right? Otherwise I'm left trying to guess what the test is doing.

Reply:
Done.